src/test/java/org/sonar/plugins/findbugs/configuration/SimpleConfiguration.java

```diff
@@ -0,0 +1,67 @@
+/*
+ * SonarQube Findbugs Plugin

```

Comment:
New files should have a SpotBugs copyright notice, not FindBugs.

Reply:
@ThrawnCA I've modified the file header but I'm not sure if that's right because I could not find a template in the project